apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/MetricsConfiguration.java

```diff
@@ -37,6 +42,20 @@ public class MetricsConfiguration extends ConfigurationOptionProvider {
         .tags("added[1.22.0]")
         .buildWithDefault(true);
 
+    private final ConfigurationOption<List<Double>> customMetricsHistogramBoundaries = ConfigurationOption.builder(new ListValueConverter<>(DoubleValueConverter.INSTANCE), List.class)
+        .key("custom_metrics_histogram_boundaries")
+        .configurationCategory(METRICS_CATEGORY)
+        .description("Defines the default bucket boundaries to use for OpenTelemetry histograms.")
+        .dynamic(true)
+        .tags("added[1.36.0]", "experimental")
+        .buildWithDefault(Arrays.asList(
+            0.00390625, 0.00552427, 0.0078125, 0.0110485, 0.015625, 0.0220971, 0.03125, 0.0441942,
+            0.0625, 0.0883883, 0.125, 0.176777, 0.25, 0.353553, 0.5, 0.707107, 1.0, 1.41421, 2.0,
+            2.82843, 4.0, 5.65685, 8.0, 11.3137, 16.0, 22.6274, 32.0, 45.2548, 64.0, 90.5097, 128.0,
+            181.019, 256.0, 362.039, 512.0, 724.077, 1024.0, 1448.15, 2048.0, 2896.31, 4096.0, 5792.62,
+            8192.0, 11585.2, 16384.0, 23170.5, 32768.0, 46341.0, 65536.0, 92681.9, 131072.0
+        ));
```

Comment:
[minor] As those values are histogram buckets and it's a bit verbose, it could be relevant to assert that values are at least properly sorted and fallback on default if that's not the case (it could also be something worth adding to spec).